docs/source/troubleshooting/linux.rst

```diff
@@ -33,3 +33,9 @@ If screencasting fails with KMS, you may need to run the following to force unpr
 Gamescope compatibility
 -----------------------
 Some users have reported stuttering issues when streaming games running within Gamescope.
+
+Controller works on Steam but not in games
```

Comment:
I believe this applies to Windows as well, other than the last sentence.

Reply:
Maybe macOS too, other than the last sentence.

Reply:
Sadly, Sunshine doesn't support gamepads on macOS yet. There was an attempt made before but there were issues:

https://github.com/LizardByte/Sunshine/pull/756
https://github.com/reenignearcher/VirtualHID-macOS (I forked the original)